x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/behavior.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { Immutable, OperatingSystem } from '../../../../../../../common/endpoint/types';
+import { BehaviorProtectionOSes, OS } from '../../../types';
+import { ConfigForm } from '../../components/config_form';
+import { RadioButtons } from '../components/radio_buttons';
+import { UserNotification } from '../components/user_notification';
+import { ProtectionSwitch } from '../components/protection_switch';
+import { APP_ID } from '../../../../../../../common/constants';
+import { LinkToApp } from '../../../../../../common/components/endpoint/link_to_app';
+import { SecurityPageName } from '../../../../../../app/types';
+
+/** The Behavior Protections form for policy details
+ *  which will configure for all relevant OSes.
+ */
+export const BehaviorProtection = React.memo(() => {
+  const OSes: Immutable<BehaviorProtectionOSes[]> = [OS.windows];
+  const protection = 'behavior_protection';
+  return (
+    <ConfigForm
+      type={i18n.translate('xpack.securitySolution.endpoint.policy.details.behavior_protection', {
+        defaultMessage: 'Behavior Protection',
+      })}
+      supportedOss={[OperatingSystem.WINDOWS]}
+      dataTestSubj="behaviorProtectionsForm"
+      rightCorner={<ProtectionSwitch protection={protection} osList={OSes} />}
+    >
+      <RadioButtons protection={protection} osList={OSes} />
+      <UserNotification protection={protection} osList={OSes} />
+      <EuiSpacer size="m" />
+      <EuiCallOut iconType="iInCircle">
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.details.detectionRulesMessage"
+          defaultMessage="View {detectionRulesLink}. Prebuilt rules are tagged “Elastic” on the Detection Rules page."
```

Comment:
What characters are being used for the quotes around `Elastic`? They don't seem like regular double quotes. I would use double quotes and escape them:

```suggestion
          defaultMessage="View {detectionRulesLink}. Prebuilt rules are tagged \"Elastic\" on the Detection Rules page."
```

Reply:
This looks strange indeed but it is also the way it is in memory, ransomware and malware cards.